Yum/src/main/java/org/bootcamp/ApplicationProperties.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2017 JR Technologies.
+ * This file is part of Yum.
+ * 
+ * Yum is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+ * 
+ * Yum is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; 
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+ * See the GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License along with Yum. 
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.bootcamp;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+
+@Component
+@ConfigurationProperties
+public class ApplicationProperties {
+    @Value("${yum.mail.from}")
```

Comment:
@lazourion no need to use value injection with @ConfigurationProperties. Refer to http://www.baeldung.com/configuration-properties-in-spring-boot